content/community/_index.html

```diff
@@ -15,7 +15,7 @@ <h1>Help us build Carvel</h1>
                 <img src="/img/github-image.svg" />
             </div>
             <div class="content">
-                <h3><a href="{{% named_link_url "github_url" %}}">Check out Github</a></h3>
+                <h3><a href="https://github.com/vmware-tanzu/carvel.dev">Check out Github</a></h3>
```

Comment:
I think it'd be easier to maintain if we preserve the existing parameterization and update where the `github_url` is being configured: https://github.com/vmware-tanzu/carvel.dev/blob/233f0b5a57b0bc2ada91e0bc49cdc82268b9579e/config.yaml#L86